editor/gui/editor_file_dialog.cpp

```diff
@@ -1394,7 +1394,7 @@ void EditorFileDialog::_update_favorites() {
 	int current_favorite = -1;
 	for (int i = 0; i < favorited.size(); i++) {
 		bool cres = favorited[i].begins_with("res://");
-		if (cres != res) {
+		if (cres != res || !favorited[i].ends_with("/")) {
```

Comment:
I don't know if you want to do a bit of a clean up alongside this, but there is actually code below that handles files by doing nothing to them. Seems like that condition can never be reached now, so I would try to organize this loop a bit better.

Also `String name = favorited[i];` can be set at the start, since we use this value from the getgo.